app/jinja_filters.py

```diff
@@ -603,7 +603,9 @@ def map_summary_item_config(
                 item_label=block.get("item_label"),
                 item_anchor=block.get("item_anchor"),
             )
-            rows.extend(list_collector_rows)
+
+            # map_list_collector_config returns List[Dict[str, List[Any]]] but mypy expects Iterable[SummaryRow]
```

Comment:
Doesn't this suggest that the return type for this function is incorrect i.e it won't only be a list of SummaryRows?

Reply:
Yeah, I tried `-> Union[SummaryRows,  list[Dict[str, List[Any]]]]` but I still get that error. I'll have another look 👍

Reply:
Removed the type ignore. Pete and I seemed to find that if you specifically type that `rows : list = []`, resolves the issue.